pkg/xds/envoy/listeners/kafka_configurer.go

```diff
@@ -0,0 +1,41 @@
+package listeners
+
+import (
+	envoy_listener "github.com/envoyproxy/go-control-plane/envoy/api/v2/listener"
+	envoy_kafka "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/kafka_broker/v2alpha1"
+
+	"github.com/kumahq/kuma/pkg/util/proto"
+)
+
+const (
+	kafkaStatPrefix = "kstat"
+)
+
+func Kafka() FilterChainBuilderOpt {
```

Comment:
We need to convert kafka stats to have proper prometheus metrics with two labels
- TYPE
- name (from stats prefix)